trulens_eval/examples/snowflake_demo/schema.py

```diff
@@ -19,9 +19,10 @@ class ModelConfig(BaseModel):
     max_new_tokens: int = 1024
     system_prompt: str = ""
     use_rag: bool = True
-    retriever: str = "Cortex Search"
+    retriever: Optional[str] = "Cortex Search"
```

Comment:
can this be `None` or something? `Optional` means it can be `None` not that it has a default value.

Reply:
None is okay if use_rag is False right?

Reply:
I'm a bit late to the game here, but for future reference, due to the way you wrote it, even if you don't give the arg, I think it's set to the default arg you've set --- which in this case is "Cortex Search".